assessment.js

```diff
@@ -1,8 +1,8 @@
 //We'll learn about require later in the course
 var ask = require('./ask.js');
 
-var answer = 'not empty';
+var answer = 0;
 
-while (answer !== '' && answer !== 'SeCrEt') {
+while (answer <= 7 && answer >= 9) {
```

Comment:
So... would this work better?

 var ask = require('./ask.js');
 
 var answer = 0;
// var theNumber = 5;
 
 while (answer < 5 && answer >= 6) {
   answer = ask("Guess which number I'm thinking of between 1 and 10?");
 }

console.log("Yes! I was thinking of 5. Good work!");